demo/components_list.py

```diff
@@ -28,6 +28,14 @@ def components_view() -> list[AnyComponent]:
                 c.Text(text='This is a text component.'),
             ]
         ),
+        c.Div(
+            components=[
+                c.Heading(text='FormattedText', level=2),
+                c.Markdown(text='`FormattedText` can be used to change the style of the text.'),
```

Comment:
please add something like

> this is particularly useful to avoid the overhead of loading the markdown renderer, or when you're including user generated content that you don't want to escape styling.